premailer/premailer.py

```diff
@@ -510,6 +513,17 @@ def _load_external(self, url):
 
         return css_body
 
+    @staticmethod
+    def six_color(color_value):
+        """Fix background colors for Lotus Notes
+
+        Notes which fails to handle three character ``bgcolor`` codes well.
+        see <https://github.com/peterbe/premailer/issues/114>"""
+
+        # Turn the color code from three to six digits
+        retval = _short_color_codes.sub(r'#\1\1\2\2\3\3', color_value)
+        return retval
```

Comment:
Feel free to change it. I prefer setting `retval` as it makes debugging much easier later on. TBH there really should be an `assert` in there:

    assert len(retval) == 7, 'Color value ({0}) of the wrong length: {0}'.format(retval, len(retval))